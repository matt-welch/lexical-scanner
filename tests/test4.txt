WHILEwhile whileWHILE WHILE 






while IFWHILE if IF

FUNC_test FUNC_(
FUNC_1 12E+)
<<<<<
>>
FUNCKY_Code
FUNC_
