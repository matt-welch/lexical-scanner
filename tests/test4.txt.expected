1 WHILEwhile 
1 whileWHILE 
1 WHILE
8 while 
8 IFWHILE 
8 if 
8 IF
10 FUNC_test 
10 FUNC
10 _
10 (
11 FUNC 
11 _
11 1 
11 12 
11 E
11 +
11 )
12 <<
12 <<
12 <
13 >>
14 FUNCKY
14 _
14 Code
15 FUNC 
15 _
